torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -41,11 +40,15 @@ struct GraphFuser {
   bool isCuda(Node * node) {
     return node->type()->expect<TensorType>()->device() != -1;
   }
-
+  bool hasSpecialAlpha(Node * node) {
+    if(!node->hasAttribute(kalpha))
+      return false;
+    return at::Scalar(node->t(kalpha)).toDouble() != 1;
+  }
   bool isFusable(Node * node) {
     if (!node->hasType()) return false;
     if (node->kind() == kFusionGroup) return true;
-    return isSimpleMap(node) && isCuda(node);
+    return isSimpleMap(node) && !hasSpecialAlpha(node) && isCuda(node);
```

Comment:
Can you leave a TODO for implementing special alpha? It seems quite easy to implement, as far as I understand you just didn't want to spend time on it right now